modules/account-lib/test/unit/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -213,9 +217,24 @@ describe('TSS ECDSA TESTS', function () {
         // and finally signs the message using their private OShare
         // and delta share received from the other signer
 
+        const hashGenerator = (hashType?: string): Hash | undefined => {
```

Comment:
we need to generate a hash each time rather than supplying the same Hash object because Hash objects hold state and the same objects.will not work for multiple uses. This would never happen in practice because each party. would be signing separately, but we run into this issue when testing